pkg/virt-controller/services/template_test.go

```diff
@@ -3448,6 +3448,59 @@ var _ = Describe("Template", func() {
 
 		})
 
+		Context("virtiofs container qos", func() {
+			DescribeTable("should container in QOSGuaranteed group ", func(req, lim kubev1.ResourceList) {
+				kvConfig := &v1.KubeVirtConfiguration{
+					SupportContainerResources: []v1.SupportContainerResources{
+						{
+							Type: v1.VirtioFS,
+							Resources: kubev1.ResourceRequirements{
+								Requests: kubev1.ResourceList{},
+								Limits:   kubev1.ResourceList{},
+							},
+						},
+					},
+				}
+				kvConfig.SupportContainerResources[0].Resources.Requests = req
+				kvConfig.SupportContainerResources[0].Resources.Limits = lim
```

Comment:
why don't you initialize these above ?

Reply:
Because container VirtioFSContainers resource usage is not base on the vmi, it's a fix value which is hardcoded